deltachat-jsonrpc/src/api/types/message.rs

```diff
@@ -121,6 +124,8 @@ impl MessageObject {
             file_mime: message.get_filemime(),
             file_bytes,
             file_name: message.get_filename(),
+
+            download_state: message.download_state().try_into().unwrap()
```

Comment:
`.unwrap()` means it can/will panic, please only use it for quick prototyping.

Reply:
also is there any particualar reason for using `try_into` instead of just `into`? the implementation below doesn't look like it can fail?

Reply:
Good point, why is message.download_state() not always returning a download state? Because there is a Done download state, so every message should have a fitting state?

Reply:
even if there is an option it could be converted to an error and then be unpacked via `?` operator